### PR TITLE
Reformat ten recipes

### PR DESCRIPTION
**Pull request by Gareth422**
I have reformatted one recipe as an example, as per waddlesplash's instructions. If this formatting is correct, I'll do the other nine.

This is for the following Google Code-in task: https://codein.withgoogle.com/dashboard/task-instances/5330222270382080/

### CODE REVIEW

**Comment by waddlesplash** [comment]
@Gareth422 Looks like you've got trailing whitespace -- you can enable "Strip trailing whitespace" in Pe preferences (I don't recall where - "Files" maybe?)

File: media-libs/libnsbmp/libnsbmp-0.1.2.recipe [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
 REQUIRES="
 	haiku
 	"
+	
+PROVIDES_devel="
+	libnsbmp_devel = $portVersion
+	devel:libnsbmp = $portVersion
+	"
+REQUIRES_devel="
+	libnsbmp$secondaryArchSuffix == $portVersion base
```
Comment (waddlesplash) [inline media-libs/libnsbmp/libnsbmp-0.1.2.recipe]: Delete this `$secondaryArchSuffix`, there are no SECONDARY_ARCHITECTURES specified.

**Comment by waddlesplash** [comment]
Other than that, looks good; go ahead and do the other 9.

**Comment by pulkomandy** [comment]
Please keep the secondary arch suffix, the next version of Netsurf (and nsbmp) will be gcc4-only.

**Comment by Gareth422** [comment]
Alright, I'll rename the PR and do the other nine.

**Comment by Gareth422** [comment]
I've finished the ten recipes, and I've made sure that there isn't any trailing whitespace. Let me know if there are any issues.

File: dev-libs/json_c/json_c-0.12.recipe [hunk 2]
```diff
@@ -5,14 +5,17 @@ construct JSON objects in C, output them as JSON formatted strings and parse \
 JSON formatted strings back into the C representation of JSON objects.
 "
 HOMEPAGE="https://github.com/json-c/json-c/wiki"
-LICENSE="MIT"
 COPYRIGHT="
 	2009-2012 Eric Haszlakiewicz
 	2004, 2005 Metaparadigm Pte Ltd
 "
```
Comment (waddlesplash) [inline dev-libs/json_c/json_c-0.12.recipe]: Proper COPYRIGHT formatting is:
```
COPYRIGHT="XXXX-XXXX Blah Blah
    XXXX Blah blah"
```
Please update accordingly.

File: dev-libs/libsolv/libsolv-0.3.0_haiku_2014_12_22.recipe [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-srcGitRev="da2650933f85f5a30d8146bfb22af53d2620d0c7"
+COPYRIGHT="2007-2013, Novell Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
 SOURCE_URI="https://github.com/weinhold/libsolv/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="79aa7772bfe403f003e8063480ddc3d4f491b3f5d7b8cace610605fba3318150"
+SOURCE_DIR="libsolv-$srcGitRev"
 SOURCE_FILENAME="$portVersionedName.tar.gz"
-REVISION="1"
+srcGitRev="da2650933f85f5a30d8146bfb22af53d2620d0c7"
```
Comment (waddlesplash) [inline dev-libs/libsolv/libsolv-0.3.0_haiku_2014_12_22.recipe]: This must be above the SOURCE_URI -- see how the SOURCE_URI uses it.

**Comment by Gareth422** [comment]
I've addressed those two issues.

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**